test_scripts/API/AutoCompleteList/005_ATF_SetGlobalProperties_P_AutoCompleteList_ValueSpaceBefore.lua

```diff
@@ -0,0 +1,79 @@
+----------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = true, resultCode = "SUCCESS"> to mobile app
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> with spaces before values.
+-- 3. SDL respond <success = true, resultCode = "SUCCESS"> to mobile app
+----------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+
+--[[ General Precondition before ATF start ]]
+config.defaultProtocolVersion = 2
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:AutoCompleteList_SpaceBefore()
+ --mobile side: sending SetGlobalProperties request
+local cid = self.mobileSession:SendRPC("SetGlobalProperties",
+    {
+      keyboardProperties =
+      {
+        keyboardLayout = "QWERTY",
+        keypressMode = "SINGLE_KEYPRESS",
+        limitedCharacterList =
+        {
+          "a"
+        },
+        language = "EN-US",
+        autoCompleteText = "Text_1, Text_2",
+        autoCompleteList = {" SpaceBefore", " SpaceBefore"}
```

Comment:
@Aleksandriys, to check space in value there is no need of two same elements(" SpaceBefore" and " SpaceBefore"), so 1 element in array is enough for check the case